cmd/oci-unpack/main.go

```diff
@@ -101,6 +101,11 @@ func (v *unpackCmd) Run(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	if _, err := os.Stat(args[1]); os.IsNotExist(err) {
+		v.stderr.Printf("destination path %s does not exist", args[1])
+		os.Exit(1)
```

Comment:
Is this a problem?  Why don't we create the directory if it is missing?  Although I'd rather `os.Mkdir` instead of `os.MkdirAll` to avoid overreacting to typo'd paths.  This would give us the same semantics as `touch(1)` and a host of other tools.